internal/providers/conjur/provider.go

```diff
@@ -94,9 +94,8 @@ func ProviderFactory(options plugin_v1.ProviderOptions) (plugin_v1.Provider, err
 		if conjur, err = conjurapi.NewClientFromTokenFile(provider.Config, tokenFile); err != nil {
 			return nil, fmt.Errorf("ERROR: Could not create new Conjur provider: %s", err)
 		}
-	case provider.AuthnURL != "" && strings.Contains(provider.AuthnURL, "authn-k8s"):
-		// Conjur provider using authenticator
-		log.Printf("Info: Conjur provider using Kubernetes authenticator-based authentication")
+	case provider.AuthnURL != "" && (strings.Contains(provider.AuthnURL, "authn-k8s") || strings.Contains(provider.AuthnURL, "authn-jwt")):
```

Comment:
Can you use a slice (array) of valid authn types instead of calling contains() for each option?

Reply:
Or else you could move this line to a helper function with a descriptive name?